cache.go

```diff
@@ -190,8 +190,8 @@ func (c *Cache) Freshen(res *Resource, keys ...string) error {
 }
 
 func hashKey(key string) string {
-	h := sha256.New()
-	io.WriteString(h, key)
+	h := fnv.New64a()
+	h.Write([]byte(key))
```

Comment:
I would imagine a part of the speedup comes from using `Write` method directly, instead of going through [`io.WriteString`](https://gotools.org/io#WriteString) which incurs a func call and additional computation (a type assertion which fails).

Have you tried benchmarking with sha256 but rewriting it to do `h.Write([]byte(key))` as well? How does that compare?